keyboards/system76/launch_1/info.json

```diff
@@ -0,0 +1,96 @@
+{
+    "keyboard_name": "System76 Launch Configurable Keyboard (launch_1)",
+    "url": "https://system76.com/accessories/launch",
+    "layouts": {
+        "LAYOUT": {
+            "c_macro": true,
+            "filename": "keyboards/system76/launch_1/launch_1.h",
```

Comment:
```suggestion
```

Reply:
Removed what I assume is superfluous information from JSON.